src/server/game/Spells/Auras/SpellAuraEffects.cpp

```diff
@@ -3901,6 +3901,24 @@ void AuraEffect::HandleModCastingSpeed(AuraApplication const* aurApp, uint8 mode
         return;
     }
 
+    if (GetAuraType() == SPELL_AURA_MOD_CASTING_SPEED_NOT_STACK)
+    {
+        // Find stronger effect
+        int32 strongEffect = 0;
+        Unit::AuraEffectList const& castingSpeedNotStack = target->GetAuraEffectsByType(SPELL_AURA_MOD_CASTING_SPEED_NOT_STACK);
+        for (AuraEffect const* aurEff : castingSpeedNotStack)
+        {
+            if (aurEff != this && abs(strongEffect) < abs(aurEff->GetAmount()))
```

Comment:
Hey Keader <3 Question: What would happen if there is -70 effect and a +50 effect. Would that ultimately mean that the -70 will be a +70 effect?

Reply:
Nope, we still use non-abs value.
It's a copy of SpellGroup system.